README.md

```diff
@@ -232,89 +392,121 @@ Available options:
   -h, --help               Show this help text
 ```
 
-The options to be enabled/disabled can be specified while running the command.
-If any of the applicable command options wasn't tagged as enabled/disabled, then
-the question is asked during the work of the tool.
+For example, the following command preconfigures the project settings with the custom prelude `relude`, included library, executable, test stanzas (but without benchmarks), creates a private repository on GitHub with the Travis CI and AppVeyor CI integrated.
 
-For example,
-
-```
-  summon new my-project with -letgcpw without -b --prelude-package relude --prelude-module Relude
+```shell
+summon new my-project with -letgcpw without -b --prelude-package relude --prelude-module Relude
 ```
 
-creates the fully functional project which uses custom prelude `relude`, contains
-library, executable file, tests and create private repository on [github](https://github.com)
-integrated with `Travis-CI`, `AppVeyor-CI`, but benchmarks won't be attached to this one.
+### TUI [↑](#structure)
+
+#### TUI new command [↑](#structure)
+
+TODO: screenshot
+
+The TUI window opens with the pre-filled information received from a config file and command-line arguments. However, it's possible to change any field you would like.
+
+At the screenshot you can see that you'll be given a few text fields to fill in or some checkboxes to represent boolean values of the configurations. To help you fill in the correct input, there are some ⓘ info messages in the `Errors` section, which give additional details on the current field.
```

Comment:
In the screenshot, you can see ... some checkboxes that represent boolean... 

additional details ~~on~~ about the current field.